src/libcore/array.rs

```diff
@@ -30,13 +27,17 @@ use crate::slice::{Iter, IterMut};
 /// Note that the traits AsRef and AsMut provide similar methods for types that
 /// may not be fixed-size arrays. Implementors should prefer those traits
 /// instead.
+#[stable(feature = "fixed_size_array", since = "1.36.0")]
```

Comment:
I don't think T-libs signed off on stabilizing `FixedSizeArray`?

Reply:
Oops, I misread?

Reply:
I believe they wanted to stabilize just the module and `TryFromSliceError`.

Reply:
I got it, we should make them unstable?

Reply:
I believe so, yes.

Reply:
Okay, thank you!